internal/validate/helpers.go

```diff
@@ -50,32 +50,32 @@ func GetPolicyConfig(ctx context.Context, policyConfiguration string) (string, e
 		if err != nil {
 			return "", err
 		}
-		return readPolicyConfigurationFile(ctx, configFile)
+		log.Debugf("Loading %s as policy configuration", configFile)
+		return ReadWorkspaceFile(ctx, configFile)
 	} else if source.SourceIsFile(policyConfiguration) && utils.HasJsonOrYamlExt(policyConfiguration) {
 		// If policyConfiguration is detected as a file and it has a json or yaml extension,
 		// we read its contents and return it.
-		return readPolicyConfigurationFile(ctx, policyConfiguration)
+		log.Debugf("Loading %s as policy configuration", policyConfiguration)
+		return ReadWorkspaceFile(ctx, policyConfiguration)
 	}
 
 	// If policyConfiguration is not a file path, git url, or https url,
 	// we assume it's a string and return it as is.
 	return policyConfiguration, nil
 }
 
-// Read policyConfiguration file and return its contents.
-func readPolicyConfigurationFile(ctx context.Context, policyConfiguration string) (string, error) {
-	// Check if policyConfig is a file path. If so, try to read it.
-	// If successful we write that into the data.policyConfiguration var.
+// Read file from the workspace and return its contents.
+func ReadWorkspaceFile(ctx context.Context, fileName string) (string, error) {
```

Comment:
I would not mention the Workspace, here, that is a Tekton term, not a concern of the CLI, not sure about converting to `string` it, might be a good opportunity to reevaluate that

```suggestion
func ReadFile(ctx context.Context, fileName string) ([]byte, error) {
```

Reply:
Removed `Workspace` from the function name. 

I think the reason behind returning `string` instead of `[]byte` would have been returning the string data as is in-case the input is a `string` and not a file in `GetPolicyConfig`. Do you still think we should change that?